### ISSUE TITLE
Web GUI is failing to load since recent update

### ISSUE THREAD

**Issue by JHarding86** [linuxserver/docker-deluge#127]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

------------------------------

## Expected Behavior
When I visit the web server address, the web GUI should be shown

## Current Behavior
A blank white screen appears. I use a [separate client](https://github.com/RudyB/Deluge-Remote) that I have sideloaded on ios, and it seems to work just fine, but my web browser fails. If I open the console I see this error: 
![Broken](https://user-images.githubusercontent.com/12802227/145766040-38a12b0e-3d8a-49f2-9b67-9eab2b20a89c.png)

## Steps to Reproduce

1. Load the docker image tagged "latest"
2. Browse to the web address specified in your docker run command

## Environment
**OS: Ubuntu Linux**
**CPU architecture:** x86_64
**How docker service was installed:** From the official docker hub lscr.io/linuxserver/deluge

## Command used to create docker container (run/create/compose/screenshot)
```
docker run -d \
  --name=deluge \
  -e PUID=1000 \
  -e PGID=1000 \
  -e TZ=America/Denver \
  -e DELUGE_LOGLEVEL=error `#optional` \
  -p 8112:8112 \
  -p 6881:6881 \
  -p 6881:6881/udp \
  -v /home/james/delugeConfig:/config \
  -v /home/james/torrents/:/downloads \
  --restart unless-stopped \
  lscr.io/linuxserver/deluge
```

## Docker logs
```
<!--- Provide a full docker log, output of "docker logs deluge" -->
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 01-envfile: executing...
[cont-init.d] 01-envfile: exited 0.
[cont-init.d] 01-migrations: executing...
[migrations] started
[migrations] no migrations found
[cont-init.d] 01-migrations: exited 0.
[cont-init.d] 10-adduser: executing...

-------------------------------------
          _         ()
         | |  ___   _    __
         | | / __| | |  /  \
         | | \__ \ | | | () |
         |_| |___/ |_|  \__/


Brought to you by linuxserver.io
-------------------------------------

To support LSIO projects visit:
https://www.linuxserver.io/donate/
-------------------------------------
GID/UID
-------------------------------------

User uid:    1000
User gid:    1000
-------------------------------------

[cont-init.d] 10-adduser: exited 0.
[cont-init.d] 30-config: executing...
[cont-init.d] 30-config: exited 0.
[cont-init.d] 90-custom-folders: executing...
[cont-init.d] 90-custom-folders: exited 0.
[cont-init.d] 99-custom-scripts: executing...
[custom-init] no custom files found exiting...
[cont-init.d] 99-custom-scripts: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
```

**Comment by JHarding86** [comment]
I just cleared cache and hard reloaded and now I get this error:
![Broken 2](https://user-images.githubusercontent.com/12802227/145767629-d689ec09-0034-488a-ab50-d126a4a8286e.png)

**Comment by j0nnymoe** [comment]
Roll back to the previous tag for now. Same is happening on our smoke tests https://ci-tests.linuxserver.io/linuxserver/deluge/latest/index.html

**Comment by ciberboy** [comment]
I have the same issue

**Comment by j0nnymoe** [comment]
@ciberboy as I mentioned above, roll back to the previous tag.

**Comment by ciberboy** [comment]
Rollback to previous version (https://hub.docker.com/r/linuxserver/deluge/tags) worked fine for me.

Thank you

**Comment by hiyadragon** [comment]
Yeah same issue pulled the from last month and webgui is working linuxserver/deluge:amd64-2.0.3-2201906121747ubuntu18.04.1-ls127

**Comment by julio8a** [comment]
Thanks for reporting, having the same issue. Rolling back fixed for me too!

**Comment by BK-STAR77** [comment]
I have the same issue too

**Comment by antoine2tt** [comment]
Same issue here ! 

Same error as @JHarding86

**Comment by gist901** [comment]
Well that was fun troubleshooting for the past hour. Should've just come here first ;p

I can verify rolling back to previous version fixes this.

**Comment by Jasmin25** [comment]
> Well that was fun troubleshooting for the past hour. Should've just come here first ;p
> 
> I can verify rolling back to previous version fixes this.

What version did you all rollback to? 

I tried `version-2.0.3-2201906121747ubuntu18.04.1` and `2.0.3-2201906121747ubuntu18.04.1-ls128` but the UI still doesn't load.

**Comment by clupss** [comment]
Confirming I also have this same issue., 
Torrents are seeding but webui won't load.

**Comment by ciberboy** [comment]
@Jasmin25 from this page, https://hub.docker.com/r/linuxserver/deluge/tags, scroll down to previous month version. Copy the command to pull directly from this tags page

**Comment by hiyadragon** [comment]
@Jasmin25  roll back to last months tag  of linuxserver/deluge:amd64-2.0.3-2201906121747ubuntu18.04.1-ls127

**Comment by Jasmin25** [comment]
> @Jasmin25 roll back to last months tag of linuxserver/deluge:amd64-2.0.3-2201906121747ubuntu18.04.1-ls127

Thanks, this one worked, the ones I posted above did not.

**Comment by minagerges** [comment]
Below is a workaround till this is fixed in next image.
Run inside the docker:
`cp /usr/lib/python3/dist-packages/deluge/ui/web/js/deluge-all-debug.js /usr/lib/python3/dist-packages/deluge/ui/web/js/deluge-all.js`

**Comment by cas--** [comment]
Apologies I can confirm this is an overlooked packaging problem with Deluge 2.0.4. This is an unfortunate situation where minifying likely failed with the deluge-all js files leaving a zero length file that in-turn broke the fallback to the debug js files. 

I'll try to make a new release this week with the fix

**Comment by jassibacha** [comment]
> Below is a workaround till this is fixed in next image. Run inside the docker: `cp /usr/lib/python3/dist-packages/deluge/ui/web/js/deluge-all-debug.js /usr/lib/python3/dist-packages/deluge/ui/web/js/deluge-all.js`

Took me a while to figure out how to get into the docker container, but this worked like a charm as a quick-fix. Thanks!

**Comment by Ihatetotank** [comment]
> Below is a workaround till this is fixed in next image. Run inside the docker: `cp /usr/lib/python3/dist-packages/deluge/ui/web/js/deluge-all-debug.js /usr/lib/python3/dist-packages/deluge/ui/web/js/deluge-all.js`

Winner winner, chicken dinner! This worked great!

**Comment by julio8a** [comment]
> > Below is a workaround till this is fixed in next image. Run inside the docker: `cp /usr/lib/python3/dist-packages/deluge/ui/web/js/deluge-all-debug.js /usr/lib/python3/dist-packages/deluge/ui/web/js/deluge-all.js`
> 
> Took me a while to figure out how to get into the docker container, but this worked like a charm as a quick-fix. Thanks!

Nice! I can also confirmed this worked for me!

**Comment by jeffreyswiggins** [comment]
I can also confirm that 

> Below is a workaround till this is fixed in next image. Run inside the docker: `cp /usr/lib/python3/dist-packages/deluge/ui/web/js/deluge-all-debug.js /usr/lib/python3/dist-packages/deluge/ui/web/js/deluge-all.js`

Thank you so much!  This worked for me and it was driving me nuts to use the thick client and switch between my 2 instances of deluge.

**Comment by dnmaia** [comment]
> Below is a workaround till this is fixed in next image. Run inside the docker: `cp /usr/lib/python3/dist-packages/deluge/ui/web/js/deluge-all-debug.js /usr/lib/python3/dist-packages/deluge/ui/web/js/deluge-all.js`

Worked perfect here. Thank you.

**Comment by emnoze** [comment]
Passing from latest to linuxserver/deluge:2.0.3-2201906121747ubuntu18.04.1-ls128 worked for me

**Comment by cas--** [comment]
I have released 2.0.5 with fix and should be available on PPA now

**Comment by aptalca** [comment]
Published:
https://github.com/linuxserver/docker-deluge/releases/tag/2.0.5-0202112151848ubuntu18.04.1-ls132
